TombEngine/Game/effects/effects.cpp

```diff
@@ -1596,14 +1641,17 @@ void ProcessEffects(ItemInfo* item)
 	switch (item->Effect.Type)
 	{
 	case EffectType::Smoke:
-		SoundEffect(SOUND_EFFECTS::SFX_TR5_HISS_LOOP_SMALL, &item->Pose);
```

Comment:
I don't think it's a good idea to reuse SAND_LOOP slot for this. Are there any other TR3/4/5 sounds which sound like hissing steam or smoke?

Reply:
with loop enabled in sound tool it sounds good, reverted back to SFX_TR5_HISS_LOOP_SMALL